src/index.html

```diff
@@ -1,13 +1,46 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no,
+                   initial-scale=1.0, maximum-scale=1.0,
+                   minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet">
```

Comment:
wrong formatting, please fix it

![image](https://github.com/mate-academy/layout_product-cards/assets/91946036/94144eb9-b942-4936-a850-67263549f5fc)

[Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)